.github/workflows/iroha2-dev-pr.yml

```
@@ -137,3 +137,50 @@ jobs:
           file: Dockerfile
           # This context specification is required
           context: .
+
+  client-cli-test:
+    strategy:
+      matrix:
+        bin:
+          - iroha_client_cli
+          - kagami
+          - iroha
+
+    runs-on: ubuntu-latest
+    container:
+      image: alexstrokelive/iroha2-ci:pytest-nightly-2023-06-25
```

```suggestion
      image: hyperledger/iroha2-ci:nightly-2023-06-25
```

why did you modify the docker image?

---

if you need to add additional tools to the image look at `Dockerfile` in the repo

---

Yes, you are right. And I added the new tools to the `Dockerfile.build` as you can see below. But for debug goals I needed to create the future image using my own docker hub account.

---

We can update the ci-image based on the changes in `Dockerfile.build` under HL ownership even before this PR merging. With the same tag or with any new (need to come up).

---

@AlexStroke so will you then restore this change before merging the PR?